src/components/Footer.astro

```diff
@@ -45,14 +46,25 @@ const navigation = {
       me: true
     },
   ],
+  organization: [
```

Comment:
Is “organization” a meaningful new grouping? And is the newsletter organization-specific as this hierarchy might suggest?

You might consider moving both of these into product—I’d sign up for the DDEV newsletter expecting to hear about the product, for example, and the organization supports it I think. (That content could easily live on the about page.)

Reply:
I was just avoiding having a long list on one place. 

I don't think people read the about page long enough to discover anything, but would think it needs a link there anyway.

Reply:
Removed organization and put those under product.